### PR TITLE
PLAT-26058 Add in missing Jep interface methods for handling Objs

### PR DESCRIPTION
**Pull request by ihleonard-c3**

### CODE REVIEW

File: PyJC3CodeGen.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 import os
-files_to_clean = ["./src/main/c/Objects/pyjc3constructor.c","./src/main/c/Objects/pyjc3multimethod.c","./src/main/c/Objects/pyjc3class.c","./src/main/c/Objects/pyjc3object.c", "./src/main/c/Include/pyjc3constructor.h", "./src/main/c/Include/pyjc3multimethod.h", "./src/main/c/Include/pyjc3method.h","./src/main/c/Include/pyjc3class.h","./src/main/c/Include/pyjc3object.h", "./src/main/c/Objects/pyjc3method.c"]
-names = ["PyJC3Constructor", "PyJC3Object", "PyJC3Class", "PyJC3Method", "PyJC3MultiMethod"]
+files_to_clean = ["./src/main/c/Objects/pyjc3constructor.c","./src/main/c/Objects/pyjc3multimethod.c","./src/main/c/Objects/pyjc3type.c","./src/main/c/Objects/pyjc3object.c", "./src/main/c/Include/pyjc3constructor.h", "./src/main/c/Include/pyjc3multimethod.h", "./src/main/c/Include/pyjc3method.h","./src/main/c/Include/pyjc3type.h","./src/main/c/Include/pyjc3object.h", "./src/main/c/Objects/pyjc3method.c"]
+names = ["PyJC3Constructor", "PyJC3Object", "PyJC3Type", "PyJC3Method", "PyJC3MultiMethod"]
```
Comment (TomTJarosz) [inline PyJC3CodeGen.py:4]: I'm assuming you did a replace all for `PyJC3Class` -> `PyJC3Type`?

File: src/main/c/Objects/pyjc3field.c [hunk 2]
```diff
@@ -366,7 +373,7 @@ PyObject* pyjc3field_get(PyJC3FieldObject *self, PyJC3Object* pyjobject)
 //    case JCHAR_ID: {
 //        jchar ret;
 //
-//        if (self->isStatic)
+//        if (self->isConstant)
```
Comment (TomTJarosz) [inline src/main/c/Objects/pyjc3field.c:376]: same question for `isStatic` -> `isConstant`

File: PyJC3CodeGen.py (hunk 1 above)
Comment (ihleonard-c3) [reply PyJC3CodeGen.py:4]: yup

File: src/main/c/Objects/pyjc3field.c (hunk 2 above)
Comment (ihleonard-c3) [reply src/main/c/Objects/pyjc3field.c:376]: yes

**Pull request merged**